src/Umbraco.Cms.Api.Management/Umbraco.Cms.Api.Management.csproj

```diff
@@ -14,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\Umbraco.PublishedCache.HybridCache\Umbraco.PublishedCache.HybridCache.csproj" />
```

Comment:
The project is in the wrong folder, it should be in `src` but it's in the `Umbraco-CMS` folder 🙈

Reply:
What the... 🙈 
Moved 🚀